cli_reference/tkn_cli/installing-tkn.adoc

```diff
@@ -6,13 +6,25 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Use the `tkn` CLI tool to manage {pipelines-title} from a terminal. The following section describes how to install `tkn` on different platforms.
+Use the CLI tool to manage {pipelines-title} from a terminal. The following section describes how to install the CLI tool on different platforms.
 
 You can also find the URL to the latest binaries from the {product-title} web console by clicking the *?* icon in the upper-right corner and selecting *Command Line Tools*.
 
 :FeatureName: Running {pipelines-title} on ARM hardware
 include::snippets/technology-preview.adoc[]
 
+[NOTE]
+====
+Both the archives and the RPMs contain the following executables:
+
+* tkn
+* tkn-pac
+* opc
+====
+
+:FeatureName: Running {pipelines-title} with the `opc` CLI tool
```

Comment:
I might have had some trouble with using attributes within the FeatureName attribute with the tech preview snippet. I recommend verifying that this renders correctly.

Reply:
Thanks for making me aware of this potential issue. I've checked the preview and also the already published [note for ARM](https://docs.openshift.com/container-platform/4.12/cli_reference/tkn_cli/installing-tkn.html) both resolve.